### PR TITLE
move dataset filter to right

### LINKED ISSUE #385: Move datasets filter widget
Either to bottom of filter list, or to results pane as in following:

image

Your thoughts @geezacoleman ?

### PR DESCRIPTION
**Pull request by ElevnLi**
fixes #385 
Screen Shot 2021-03-24 at 3 08 49 pm

### CODE REVIEW

File: search/src/Components/wrapper/reactive_search.js [hunk 1]
```diff
@@ -627,6 +618,15 @@ class ReactiveSearchComponent extends Component {
                 </div>
                 <div style={{ position: "absolute", left: "20rem", paddingRight: "1rem" }}>
                     <SelectedFilters clearAllLabel="Clear filters" />
+                    <MultiDropdownList
+                        componentId="dataset_name_filter"
+                        dataField="dataset_name.keyword"
+                        title="Dataset Name"
```
Comment (jnothman) [inline search/src/Components/wrapper/reactive_search.js:624]:
```suggestion
                        title="Datasets"
```

(I did think about making this dynamic, e.g. `title={{numberOfDatasets} + " Datasets"}`, but not sure of how to extend the render hook on MultiDropdownList)

**Comment by jnothman** [comment]
Is there enough margin between this and images below?

**Comment by ElevnLi** [comment]
> Is there enough margin between this and images below?

Do we need more room?

**Comment by ElevnLi** [comment]
> Is there enough margin between this and images below?

Do we need more margin?
Screen Shot 2021-03-24 at 4 45 15 pm

**Comment by jnothman** [comment]
Maybe leave a little more vertical room, and change "x results found" to "x annotated images found"

**Comment by ElevnLi** [comment]
Screen Shot 2021-03-24 at 5 30 47 pm

**Comment by jnothman** [comment]
Nice.

**Review by jnothman — APPROVED** [review]

**Comment by jnothman** [comment]
ship it!

**Comment by jnothman** [comment]
This PR left the filter on the left in! Whoops...

**Comment by jnothman** [comment]
Sorry, mut have been a merge error on my part

**Pull request merged**